spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/core/mapping/DefaultNeo4jPersistentEntity.java

```diff
@@ -171,13 +178,23 @@ private void verifyDynamicAssociations() {
 	private String computePrimaryLabel() {
 
 		Node nodeAnnotation = this.findAnnotation(Node.class);
-		if (nodeAnnotation == null || nodeAnnotation.labels().length != 1) {
+		if (nodeAnnotation == null || nodeAnnotation.labels().length < 1) {
 			return this.getType().getSimpleName();
 		} else {
 			return nodeAnnotation.labels()[0];
```

Comment:
We stick with the first label being the primary? 

That's just a question, the answer should go into the documentation

Reply:
Since we do not have distinct parameters I would suggest doing this. Will add this to the documentation with this PR.